frontend/public/components/marketplace/modal-overlay.jsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from 'patternfly-react/dist/esm/components/Button/Button';
+import Modal from 'patternfly-react/dist/esm/components/Modal/Modal';
+import CatalogItemHeader from 'patternfly-react-extensions/dist/esm/components/CatalogItemHeader/CatalogItemHeader';
+import PropertiesSidePanel from 'patternfly-react-extensions/dist/esm/components/PropertiesSidePanel/PropertiesSidePanel';
+import PropertyItem from 'patternfly-react-extensions/dist/esm/components/PropertiesSidePanel/PropertyItem';
+
+class MarketplaceModalOverlay extends React.Component {
+  render() {
+    const {item, close /* openSubscribe */} = this.props;
+    const {tileName, tileImgUrl, tileProvider, tileDescription, version, certifiedLevel, healthIndex, repository, containerImage, createdAt, support} = item;
+    const notAvailable = <span className="properties-side-panel-pf-property-label">N/A</span>;
+    return (
+      <React.Fragment>
+        <Modal show={true} className="right-side-modal-pf" bsSize={'lg'}>
+          <Modal.Header>
+            <CatalogItemHeader
+              className="co-marketplace-modal__item-header"
+              iconImg={tileImgUrl}
+              title={tileName}
+              vendor={<span> {tileProvider}</span>}
+            />
+            <Modal.CloseButton onClick={close} />
+          </Modal.Header>
+          <Modal.Body>
+            <div className="co-marketplace-modal__body">
+              <PropertiesSidePanel>
+                <Button bsStyle="primary" className="co-marketplace-modal__subscribe" /* onClick={ openSubscribe }*/ >
+                  Subscribe
+                </Button>
+                <PropertyItem label={'Operator Version'} value={version || notAvailable} />
```

Comment:
I don't think you need the `{}` syntax here? Does `label="Operator Version"` work?